src/core/Button/style.ts

```diff
@@ -30,13 +47,26 @@ const ButtonBase = styled(Button, {
 
     const padding = variant === "outlined" ? outlinedPadding : containedPadding;
 
+    const border = borders && borders[colorProp];
+    const color = colors && colors[colorProp];
+
+    if (!border) {
+      throw new Error(`Button border not found: ${colorProp}`);
+    }
+    if (!color) {
+      throw new Error(`Button color not found: ${colorProp}`);
+    }
+
     return `
+      background-color: ${color[400]};
+      border: ${border[400]};
+      color: white;
```

Comment:
@tihuan, adding these attributes seems to be breaking the secondary button, as that is no longer styled correctly (it ends up getting filled so it just looks like the primary button).  I think we may need to go with just creating a separate override style for the error button.  I don't know if I'll have time to give that a shot, but maybe we can sync up early next week?